lib/components/narrative/default/default-itinerary.js

```diff
@@ -141,39 +158,46 @@ const ITINERARY_ATTRIBUTES = [
 
 class DefaultItinerary extends NarrativeItinerary {
   _onMouseEnter = () => {
-    const {active, index, setVisibleItinerary, visibleItinerary} = this.props
+    const { active, index, setVisibleItinerary, visibleItinerary } = this.props
     // Set this itinerary as visible if not already visible.
-    const visibleNotSet = visibleItinerary === null || visibleItinerary === undefined
-    const isVisible = visibleItinerary === index ||
-      (active === index && visibleNotSet)
+    const visibleNotSet =
+      visibleItinerary === null || visibleItinerary === undefined
+    const isVisible =
+      visibleItinerary === index || (active === index && visibleNotSet)
     if (typeof setVisibleItinerary === 'function' && !isVisible) {
-      setVisibleItinerary({index})
+      setVisibleItinerary({ index })
     }
-  }
+  };
```

Comment:
Remove extra semicolons in this file.

Reply:
I ran prettier at some point which caused these changes. Removed in 513973b